server/remote/msgraph/delete_calendar.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package msgraph
+
+func (c *client) DeleteCalendarByID(calendarID string) error {
+	// TODO: Implement
```

Comment:
what else is there to implement?

Reply:
I introduced the method, but haven't actually tested it.